x/distribution/types/delegator_info.go

```diff
@@ -49,9 +57,36 @@ func (di DelegationDistInfo) WithdrawRewards(wc WithdrawContext, vi ValidatorDis
 
 	accum := di.GetDelAccum(wc.Height, delegatorShares)
 	di.DelPoolWithdrawalHeight = wc.Height
-	withdrawalTokens := vi.DelPool.MulDec(accum).QuoDec(vi.DelAccum.Accum)
```

Comment:
One of the bug fixes

Reply:
opened up an issue for this bug... still kinda confused as to why Dec would react this way - it may be the expected behaviour but it just seems funky - https://github.com/cosmos/cosmos-sdk/issues/2888